official/resnet/keras/keras_imagenet_main.py

```diff
@@ -87,7 +87,9 @@ def run(flags_obj):
   Raises:
     ValueError: If fp16 is passed as it is not currently supported.
   """
-  if flags_obj.enable_eager:
+  # TODO(tobyboyd): Remove eager flag when tf 1.0 testing ends.
+  # Eager is default in tf 2.0 and should not be toggled
+  if flags_obj.enable_eager and not keras_common.is_v2_0():
```

Comment:
can we add the same check in keras_cifar_main?

Reply:
I did see above.  :-)  Unless I do not understand you comment.

Reply:
oh my bad, that's exactly what I meant. somehow didn't see that file.